### ISSUE TITLE
statusServer2: sorting on status broken

### ISSUE THREAD

**Issue by jbrzusto** [jbrzusto/motusServer#291]
API reply has this error:
```
Error in rsqlite_send_query(conn@ptr, statement) : 
  no such column: t1.sjDone
```
looks like a typo:  `isDone` misspelled as `sjDone`

**Comment by jbrzusto** [comment]
no, it's supposed to be sjDone (subjob done), but isn't being handled correctly.

**Comment by jbrzusto** [comment]
need to fix the logic in the query:
- `sjDone` is generated in the grouping, so should not be prefixed by `t1.`
- need to use `having` to select appropriate groups on `sjDone`
- should not end up with a clause like `sjDone  > 1`, since that never includes records.
i.e. "done" needs to be treated differently from the other sort columns

**Comment by jbrzusto** [comment]
closed by 587907b61b7ec78f57b9f66c6a09eb6365a79719
